### PR TITLE
chore: Correct jsonl validation

### PR DESCRIPTION
**Pull request by isaac-chung**
Previous PR did not actually validate anything. 
Added logging to confirm the validation works now as intended. (also by changing one of the keys and getting an error)

### CODE REVIEW

**Review by KennethEnevoldsen — COMMENTED** [review]
Will you create a faulty JSONL just to see that the CI fails

File: docs/mmteb/points/432.jsonl [hunk 1]
```diff
@@ -1,2 +1,3 @@
 {"GitHub": "asparius", "New dataset": 6}
-{"GitHub": "isaac-chung", "Review PR": 2}
\ No newline at end of file
+{"GitHub": "isaac-chung", "Review PR": 2}
+{"GitHub": "isaac-chung", "xxx": 2}
```
Comment (isaac-chung) [inline docs/mmteb/points/432.jsonl:3]: faulty key in jsonl file

**Comment by isaac-chung** [comment]
Something's not right. Let me debug.

**Comment by isaac-chung** [comment]
@KennethEnevoldsen got it to fail now 👍 going to remove the line in the jsonl file.

**Review by KennethEnevoldsen — APPROVED** [review]

**Comment by KennethEnevoldsen** [comment]
Seems like it was a good thing to get it to fail. Thanks for this PR, will merge it in

**Comment by isaac-chung** [comment]
Yes good call. 

I didn't include points here. Would this count towards something like "Bug/fixes" / how many?

**Comment by KennethEnevoldsen** [comment]
I would add 2 points in coordination (coordination contributions for mmteb by auto validating points)

**Pull request merged**